example/src/main.ts

```diff
@@ -387,6 +393,64 @@ export const createMynahUI = (initialData?: MynahUIDataModel): MynahUI => {
     onInfoLinkClick: (tabId, link, mouseEvent) => {
       Log(`Link inside prompt info field clicked: <b>${link}</b>`);
     },
+    upDownArrowKeyPress(tabId: string, direction: 'up' | 'down') {  
+      // reset promptInputText and any possible attachment/commands from previous prompts
+      mynahUI.updateStore(tabId, {
+        promptInputText: '',
+      });
+      mynahUI.clearPrompt(tabId);
+
+      const tab = mynahUI.getAllTabs()[tabId];
```

Comment:
rename to `currentTab` for clarity. This is usually our convention when we grab the current tab.